en/restart-a-tidb-cluster.md

```diff
@@ -6,13 +6,68 @@ category: how-to
 
 # Restart a TiDB Cluster in Kubernetes
 
-If you find that the memory leak occurs in a Pod of the TiDB cluster during use, you need to restart the cluster. This document describes how to logoff a Pod and restart a TiDB cluster gracefully.
+If you find that the memory leak occurs in a Pod during use, you need to restart the cluster. This document describes how to gracefully rolling restart all the Pods in a component of the TiDB cluster, or gracefully logoff a Pod in the TiDB cluster and then restart the Pod using the gracefully restart command.
```

Comment:
```suggestion
If you find that the memory leak occurs in a Pod during use, you need to restart the cluster. This document describes how to gracefully perform a rolling restart to all the Pods in a component of the TiDB cluster, or gracefully log off a Pod in the TiDB cluster and then restart the Pod using the graceful restart command.
```

Reply:
How about `perform a graceful rolling restart`? I found in other sources that `graceful restart` is usually put together rather than split up.

Reply:
Comment changed.